roles/backend/tasks/main.yml

```diff
@@ -9,6 +9,9 @@
   template:
     src: backend.yml.j2
     dest: /etc/sensu/backend.yml
+    owner: &sensu_user sensu
+    group: &sensu_group sensu
```

Comment:
Do we intend to use user and group values anywhere else? If no, then we can safely drop the anchors. Also, can we add a short comment about the hardcoded user and group? Something like *keep this in sync with the user an group that the backend service is running as*, just to give future us some hints.

Reply:
Actually, we will use them later, but that doesn't mean that we have to have them here. I'll just drop this for this particular commit, because the anchors will have moved up anyway.
I'll add a comment.